app/models/miq_report.rb

```diff
@@ -242,6 +242,16 @@ def self.display_name(number = 1)
     n_('Report', 'Reports', number)
   end
 
+  def userid=(_userid)
+    # there are no MiqReport.userid and MiqReport.group_description attributes and after adding 'userid'
```

Comment:
This can be simplified to something like -
`# Stubbed method to handle 'userid' attr that may be present in the exported hash which does not exist in the User class`

Reply:
corrected